setup.py

```diff
@@ -16,7 +16,7 @@
     packages=find_packages(exclude=('tests.*', 'tests', 'example')),
     install_requires=[
         'Django>=1.8',
-        'sqlparse',
+        'sqlparse>=0.2.0',
```

Comment:
(And I'm especially not a fan of shaming badges that become red if I don't look at the repo for a month or two.)